go-runtime/compile/schema.go

```diff
@@ -579,3 +583,16 @@ func tokenFileContainsPos(f *token.File, pos token.Pos) bool {
 	base := f.Base()
 	return base <= p && p < base+f.Size()
 }
+
+func maybeExtractStructTag(fieldTags string, tag string) string {
+	for _, t := range strings.Split(fieldTags, ",") {
```

Comment:
Use [reflect.StructTag](https://pkg.go.dev/reflect@go1.21.6#StructTag) here instead of parsing this ourselves. Something like `reflect.StructTag(fieldTags).Get(aliasFieldTag)` should suffice.

Reply:
StructTag will correctly handle escaping for us.

Reply:
ah nice ty! [much cleaner](https://github.com/TBD54566975/ftl/pull/836)